vslib/SwitchStateBase.cpp

```diff
@@ -4149,6 +4321,51 @@ sai_status_t SwitchStateBase::queryAttrEnumValuesCapability(
 
     return SAI_STATUS_NOT_SUPPORTED;
 }
+
+sai_status_t SwitchStateBase::queryMacsecPostCapability(
+                              _In_ sai_object_type_t object_type,
+                              _Out_ sai_attr_capability_t *attr_capability)
+{
+    SWSS_LOG_ENTER();
+
+    attr_capability->create_implemented = false;
+    attr_capability->set_implemented    = false;
+    attr_capability->get_implemented    = false;
+
+    std::ifstream ifs;
+    ifs.open(VS_SAI_FIPS_POST_CONFIG_FILE);
```

Comment:
this file path is hardcoded, no way to change it, all configs are passed via config.ini

Reply:
Hi @kcudnik , the file is used to  configure and test various SAI POST  behaviors.  So we should never need to change its name once created. 

I searched `config.ini` in sairedis repo, and it seems it's only used in proxylib https://github.com/sonic-net/sonic-sairedis/blob/master/proxylib/Proxy.cpp#L129.  I am not sure this is the same file you referred to. Can you point me to an example of passing config to VS via `config.ini`?

Reply:
@kcudnik  gentle reminder, thx